package/MDAnalysis/core/selection.py

```diff
@@ -794,28 +776,55 @@
         return mask
 
 
+class BoolSelection(Selection):
+    """Selection for boolean values"""
+
+    def __init__(self, parser, tokens):
+        values = grab_not_keywords(tokens)
+        if not values:
+            values = [True]
+        val = values[0]
+        if isinstance(val, str):
+            val = val.lower()
+            if val == "false":
+                val = False
+            elif val == "true":
+                val = True
```

Comment:
Else? Ie what happens for bad strings here? Are they caught prior?

Reply:
An informative error is now raised.